sw/device/lib/testing/pwrmgr_testutils.h

```diff
@@ -0,0 +1,20 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_PWRMGR_TESTUTILS_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_PWRMGR_TESTUTILS_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/dif/dif_pwrmgr.h"
+
+/**
+ * Set the device in low power mode. All clocks and power domains are turned
+ * off. Consider adding a parameter to control them. A WFI instruction needs to
```

Comment:
nit: Should "Consider adding a parameter to control them."  be a TODO (or an issue)?